### PR TITLE
Fix ngram link

### PR DESCRIPTION
**Pull request by markcmiller86**
I discovered the ngram link wasn't working and adjusted it to use a space char (`%20`) in the URL instead of a dash (`-`).

### CODE REVIEW

File: Articles/Blog/InclusiveLanguageCodeUpdateExample.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ One way software projects can address inclusion is to identify and improve langu
...
+Examination of [Google's NGram viewer](https://books.google.com/ngrams/graph?content=master%20slave&year_start=1800&year_end=2019&case_insensitive=on&corpus=26&smoothing=3&direct_url=t4%3B%2Cmaster%20-%20slave%3B%2Cc0%3B%2Cs0%3B%3Bmaster%20-%20slave%3B%2Cc0%3B%3BMaster%20-%20Slave%3B%2Cc0%3B%3BMaster%20-%20slave%3B%2Cc0%3B%3BMASTER%20-%20SLAVE%3B%2Cc0) shows a sharp rise in its use beginning with [Raymond Geortz's](https://en.wikipedia.org/wiki/Raymond_Goertz) description of a [remote manipulator](https://en.wikipedia.org/wiki/Remote_manipulator), which is a sort of robotic version of a [glove box](https://en.wikipedia.org/wiki/Glovebox).
```
Comment (bernhold) [inline Articles/Blog/InclusiveLanguageCodeUpdateExample.md:13]: Mark when I put this in preview and tried it, the link I got was <https://books.google.com/ngrams/graph?content=master%20slave&year_start=1800&year_end=2019&case_insensitive=on&corpus=26&smoothing=3&direct_url=t4%3B%2Cmaster%20-%20slave%3B%2Cc0%3B%2Cs0%3B%3Bmaster%20-%20slave%3B%2Cc0%3B%3BMaster%20-%20Slave%3B%2Cc0%3B%3BMaster%20-%20slave%3B%2Cc0%3B%3BMASTER%20-%20SLAVE%3B%2Cc0> and it came up with *nothing*.  So something is still not right.

File: Articles/Blog/InclusiveLanguageCodeUpdateExample.md [hunk 2]
```diff
@@ -168,7 +168,7 @@ That ended up being a good thing since it prompted us to open up the conversatio
...
+While there was [much discussion](https://www.google.com/search?q=%22master%22+%22slave%22+terminology+in+software&sxsrf=AOaemvIy5OkrC1xJEV_CJrUNIG3Tvmtlrg:1643308409486&source=lnt&tbs=qdr:y&sa=X&ved=2ahUKEwjdlYPwyNL1AhVuD0QIHXqoBOsQpwV6BAgBEBk&biw=1616&bih=948&dpr=1) of the `master/slave` issue in the media throughout 2020, it has been a [known](https://datatracker.ietf.org/doc/html/draft-knodel-terminology-00) and [long-standing](https://www.jstor.org/stable/40061475) problem, touched on far before 2020, with the earliest online mentions going back [nearly 20 years](http://www.cnn.com/2003/TECH/ptech/11/26/master.term.reut/index.html), and other public discourse almost certainly much earlier.
```
Comment (bernhold) [inline Articles/Blog/InclusiveLanguageCodeUpdateExample.md:171]: This change worked fine.

**Review by bernhold — COMMENTED** [review]
Please see comments inline.

File: Articles/Blog/InclusiveLanguageCodeUpdateExample.md (hunk 1 above)
Comment (markcmiller86) [reply Articles/Blog/InclusiveLanguageCodeUpdateExample.md:13]: Hmm...that links works for me...but the one in the `.md` file and the one you pasted into the coversation above. Do you have a filter blocking your access?

Comment (markcmiller86) [reply Articles/Blog/InclusiveLanguageCodeUpdateExample.md:13]: Sorry...`but`==>`both`

**Comment by markcmiller86** [comment]
> first link is not working for me

It does work for me. We need a third party to decide which if our outcomes is the unique one.

**Comment by bernhold** [comment]
I should have been clearer.  The link works in the sense that it seems to be a valid NGram Viewer query.  But it returns no results.  I'm attaching a screenshot.
Capture

**Comment by markcmiller86** [comment]
Ok, I get this...

Screen Shot 2022-09-28 at 1 14 24 PM

Did you try doing anything to cause browse to refresh the page/screen? Maybe adjust the text in search field a bit too?

**Comment by bernhold** [comment]
I couldn't get anything out of tweaking from your URL as a starting point.  However going into the ngram tool from scratch I got something to work with this URL <https://books.google.com/ngrams/graph?content=master+slave&year_start=1800&year_end=2019&corpus=en-2019&smoothing=3&case_insensitive=true>

**Comment by markcmiller86** [comment]
That works for me. I'll push a change with that newer link.

**Comment by bernhold** [comment]
@markcmiller86 just FYI, the person you tagged in your commit is not me.  My preferred username in most contexts does not include the t.  Which is confusing, I know, but it harks back to my first unix account, back in the days when usernames had at most 8 characters -- just like Fortran variable names.  Ah, the good old days!

**Review by bernhold — APPROVED** [review]
Works for me now.  I'm going to merge and rebuld.

**Comment by markcmiller86** [comment]
> @markcmiller86 just FYI, the person you tagged in your commit is not me

Yeah. Saw that and tried to correct. Unfort...it was not in a comment. Its in the commit message itself and re-writing it at this point required rebases and other things I didn't feel comfortable trying. Sorry to whoever I wound up spaming.

**Pull request merged**